packit_service/worker/reporting.py

```diff
@@ -254,8 +256,11 @@ def report_status_by_comment(
     def get_statuses(self):
         self.project_with_commit.get_commit_statuses(commit=self.commit_sha)
 
-    def _has_identical_comment(self, body: str, mode: DuplicateCheckMode) -> bool:
+    def _pr_has_identical_comment(self, body: str, mode: DuplicateCheckMode) -> bool:
         """Checks if the body is the same as the last or any comment based on mode."""
+        if not self._packit_user:
+            raise PackitException("packit_user is required for identical comment check")
```

Comment:
I am just wondering if there isn't a better way of doing this. Currently the exception is not caught anywhere so it would go to the user, I assume, even though it is just our internal error caused by the fact that we forgot to set `packit_user` in some environment (e.g. hardly in this case). And I can see that sometimes in the future, someone is going to add `status_reporter.comment(..., duplicate_check=DuplicateCheckMode.check_last_comment)` without realizing that it is not valid. I don't really see a good way now

Reply:
Yes, I realized that as well. I'll properly set it in hardly.